Update base OS info #268 (#298)
rockstor/rockstor-doc: installation/quickstart.rst

hardware or as a VM with the following minimum system requirements and
recommendations.

* 64-bit Intel/AMD, or ARM64 processor (2+ cores recommended).
* 2GB RAM (4GB+ recommended).
* 16GB drive for Rockstor's system disk; if USB key use only fast variants
  (32GB+ SSD recommended).
* One or more additional drives for data use only (less than 1GB ignored, 5GB+
  recommended).
* Ethernet interface (with internet access for updates).
* All drives must have unique serial numbers (real drives do); not all VM [*]_
  systems default to this.
* A UPS (recommended) that is supported by `NUT <https://networkupstools.org/>`_.
* USB port and 1GB+ USB key for the x86_64 installation media; or a DVD drive.
  Rasbperry Pi 4 / RPi 400 installs are via a self expanding system disk image,

>   and so do not required separate install media.

Reviewer: ```suggestion
  and so do not require a separate install media.
```

Command line options
~~~~~~~~~~~~~~~~~~~~

To create a USB install disk on Linux or Mac one can use the dd command.
For example if your USB device is /dev/sdc then from within the directory

> containing your downloaded or self built

Reviewer: ```suggestion
containing your downloaded or self-built
```

Note that the installer file name will vary depending on the Rockstor
installer profile used: i.e. the base OS version, the general or machine
specific nature, i.e.: "generic", "ARM64EFI", or specific Pi4; and the
target architecture. If the suggested edits were not performed during the DIY
installer builder method then the file name & required single line command
will be more like:

    dd if=Rockstor-NAS.x86_64-4.0.7-0.install.iso of=/dev/sdc

**The Pi4 specific installer**, when downloaded, is a raw.zx image of a self
expanding system disk.
To transfer this file to the example proposed system disk of /dev/sdc the
following single line command could be used:

    xzcat Rockstor-Leap15.2-RaspberryPi4.aarch64-4.0.7-0.raw.xz | dd bs=4M of=/dev/sdc iflag=fullblock conv=notrunc status=progress

If you **built your own Pi4 installer** via our
`rockstor-installer <https://github.com/rockstor/rockstor-installer>`_
instructions then you can forgo the initial xzcat extraction step and use
(single line command):

    dd bs=4M if=Rockstor-Leap15.2-RaspberryPi4.aarch64-4.0.7-0.raw of=/dev/sdc iflag=fullblock conv=notrunc status=progress

> For the more technically interested we create our xz download image files from

Reviewer: ```suggestion
For the more technically interested, we create our xz download image files from
```

the raw installer created by the `kiwi-ng <https://github.com/OSInside/kiwi>`_
system that our
`rockstor-installer <https://github.com/rockstor/rockstor-installer>`_ is a
configuration for via "xz --threads=4 --memlimit-compress=80% Rockstor-...raw"
to enable multi-threaded decompress.

**The ARM64EFI generic images**, when downloaded, are available in both the
raw.zx file format, like the Pi4 images, and in pre-sized (16GB) qcow2 formats.

> For the raw.zx downloaded files the single line command is identical to the Pi4

Reviewer: ```suggestion
For the raw.zx downloaded files, the single line command is identical to the Pi4
```
